### ISSUE TITLE
postman: `failed to execvp` multiple standard binaries (`cat`, `xdg-settings`, `xdg-open`)

### ISSUE THREAD

**Issue by Tchekda** [NixOS/nixpkgs#325307]
### Describe the bug
Using the latest version (b88607fa3d5e17739aa01c8c0449ad3b8dee7ab4) of the derivation, I am getting an error when starting Postman and when trying to open the browser from Postman (for login).
Here are the errors: 
```
LaunchProcess: failed to execvp:
xdg-settings
```
```
LaunchProcess: failed to execvp:
xdg-open
```
### Steps To Reproduce
1. Install the package
2. Launch the app
3. Try to login
4. No browser is opened and the error is printed in the system journal

### Expected behavior
Postman should have access to the xdg environment to allow integrations with browser and other apps

### Additional context
This error wasn't happening in the nixos-23.05 version of the package.

If I change the last line of `postFixup`
https://github.com/NixOS/nixpkgs/blob/ab82a9612aa45284d4adf69ee81871a389669a9e/pkgs/development/web/postman/linux.nix#L159
It seems to be working again, meaning the fact that PATH is overriden has an impact
```sh
wrapProgram $out/bin/postman --prefix PATH : ${lib.makeBinPath [ openssl ]}
```

### Notify maintainers
@johnrichardrinehart @evanjs @tricktron @Crafter

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
- system: `"x86_64-linux"`
 - host os: `Linux 6.1.73, NixOS, 23.11 (Tapir), 23.11.3019.8bf65f17d807`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.18.1`
 - channels(tchekda): `"android-nixpkgs, home-manager-23.11.tar.gz, home-manager-master, nixos-23.11, nixos-21.11-21.11, nixos-unstable, unstable"`
 - channels(root): `"home-manager-23.11.tar.gz, home-manager-master, nixos-23.11, nixos-21.11-21.11, nixos-hardware, nixos-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

**Comment by amirsalarsafaei** [comment]
I don't think this is a good fix up,.I think xdg-utils should be in the Deps that was my fixup if everyone agrees I can draft a pull request
